src/proficiency.h

```diff
@@ -12,6 +12,7 @@
 #include "color.h"
 #include "flat_set.h"
 #include "optional.h"
+#include "options.h"
```

Comment:
If the stuff from options.h is only used in the cpp file it's best to move it there, so it doesn't get included everywhere proficiency.h is included